src/abbreviate.js

```diff
@@ -8,6 +8,11 @@ import {format} from "d3-format";
 */
 export default function(n) {
   if (typeof n !== "number") return "N/A";
+  let isNegative;
+  if (n < 0) {
+    isNegative = true;
+    n *= -1;
+  }
   const length = n.toString().split(".")[0].length;
```

Comment:
can't this be solved simply by modifying the `length` variable to remove hyphens?

```js
const length = n.toString().split(".")[0].replace("-", "").length;
```

Reply:
ahh, it sure can! will make that change.